.ci/.docker-images.yml

```diff
@@ -16,6 +16,8 @@ templates:
     name: "apm-agent-python"
     repository: "elastic/apm-agent-python"
     working_directory: "tests"
+    build_script: docker build --build-arg PYTHON_IMAGE=${TAG/-/:} -t elasticobservability/apm-agent-python:${TAG} .
```

Comment:
Just to make sure nobody mistakes these images for actual release artifacts of the agent itself, we could rename them to something like `apm-agent-python-testing`